src/app.rs

```diff
@@ -176,76 +152,86 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
                 .unwrap_or_default(),
         );
 
-        let app_init_cfg = AppInitCfg {
```

Comment:
This variable, and its use, made removing `AppInitCfg` non-trivial. I ended up removing the abstraction layers, and using the implementations directly.